README.md

```diff
@@ -0,0 +1,3 @@
+# AMITY
+
```

Comment:
add more details to the readme. How to run the program, screenshots and how to run the test. Add CI badges to the Readme